dasharo-compatibility/nvme-support.robot

```diff
@@ -60,7 +60,7 @@ NVM001.301 NVMe support in OS (Windows)
     Should Contain    ${out}    DiskDrive
     # Exit from root user
 
-NVM001.401 NVMe support (ESXi)
+NVM001.401 NVMe support in OS (ESXi)
     [Documentation]    Verify that ESXi is installed and booted from an NVMe drive.
     ...    Check that NVMe is detected and marked as the boot device.
```

Comment:
https://github.com/Dasharo/open-source-firmware-validation/blob/test-ids-fix/docs/adding-and-naming-test-cases.md#transitioning, here and in all other test cases. Even though it wasn't changed in OSFV as these are newly added tests, IDs for them existed and were used in releases.

Basically, everything that has `changed_to` in JSON, must have a matching `Previous IDs`, in the future it will be checked by CI, but we need to clean up first.